common/springUtilities/teamFunctions.lua

```diff
@@ -107,6 +107,18 @@ local function getSettings()
 	return settings
 end
 
+function GetScavTeamID()
```

Comment:
Unrelated change

Reply:
sorry that was for a different issue, how do i exclude it from this pr?